packages/commons/src/eservice-documents/eserviceDocumentUtils.ts

```diff
@@ -105,37 +105,34 @@ export const interpolateApiSpec = async (
     contentType: string;
     prettyName: string;
   },
-  eserviceInstanceInterfaceData: {
-    contactName?: string;
-    contactEmail?: string;
-    contactUrl?: string;
-    termsAndConditionsUrl?: string;
-    serverUrls: string[];
-  }
+  serverUrls: string[],
+  eservieserviceInstanceInterfaceDataRest:
```

Comment:
Align name with the param of the other function?  `eserviceInstanceInterfaceRestData`